roles/openldap/templates/phpldapadmin.service

```diff
@@ -10,7 +10,7 @@ ExecStartPre=-/usr/bin/docker kill %n
 ExecStartPre=-/usr/bin/docker rm %n
 ExecStartPre=/usr/bin/docker pull osixia/phpldapadmin:0.7.2
 ExecStart=/usr/bin/docker run -i --name %n --network host \
-    --env PHPLDAPADMIN_LDAP_HOSTS={{ uri_ldap }} \
+    --env PHPLDAPADMIN_LDAP_HOSTS={{ ldap_uri | regex_replace('^ldaps?://','')}} \
```

Comment:
Not sure, but shouldn't this be "ldap" instead of "ldaps" in the regex pattern?

Reply:
The `?` after the `s` from `ldaps` in the regex means match zero or one time, so the regex will match both `ldap://` as well as `ldaps://` and we do not need separate `ldap_uri` and `ldaps_uri` variables anymore: just `ldap_uri` is enough and can use both plain and encrypted ldap traffic.

Reply:
Ok, thanks for the explanation :-)